webapp/src/models/MessageMonitorNotification.d.ts

```diff
@@ -0,0 +1,12 @@
+type MessageMonitorNotification =
+  | IntersectionReferenceAlignmentNotification
+  | SignalGroupAlignmentNotification
+  //   | SpatMinimumDataNotification
```

Comment:
I assume these are commented for a reason?

Reply:
Changed to a TODO comment